.github/workflows/build-mudlet.yml

```
@@ -45,13 +45,24 @@ jobs:
             qt: '5.14.1'
             deploy: 'deploy'
 
+    env:
+      BOOST_ROOT: ${{github.workspace}}/3rdparty/boost
+      BOOST_URL: https://sourceforge.net/projects/boost/files/boost/1.72.0/boost_1_72_0.tar.bz2/download
+
+
     steps:
+    - name: Checkout Mudlet source code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+
     - name: Restore Qt cache
-      uses: actions/cache@v2.1.4
+      uses: actions/cache@v2
       id: cache-qt
       with:
         path: ${{runner.workspace}}/Qt/${{matrix.qt}}
-        key: ${{matrix.os}}-qt-${{matrix.qt}}
+        key: ${{matrix.os}}-qt-${{matrix.qt}}-deletmee
```

Is that a typo - should it not be:
```suggestion
        key: ${{matrix.os}}-qt-${{matrix.qt}}-deleteme
```